src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Ball {
+
+  private String color;
+  private int number;
+
+  public String getColor() {
+    return color;
+  }
+
+  public void setColor(String color) {
+    this.color = color.toLowerCase();
+  }
+
+  public int getNumber() {
+    return number;
+  }
+
+  public void setNumber(int number) {
+    this.number = number;
+  }
+
+  public Ball(String color, int number){
+    this.color = color;
+    this.number = number;
+  }
```

Comment:
It's good practice to use access modifiers for constructors as well. Consider adding an access modifier to the constructor to specify its visibility explicitly.